### ISSUE TITLE
Clang appears llvm.ident in Chapel LLVM IR

### ISSUE THREAD

**Issue by mppf** [chapel-lang/chapel#11249]
When compiling Chapel programs with --llvm, the LLVM IR shows llvm.ident and debuginfo producer of clang. It does this because clang actually produces the LLVM module Chapel works with initially, and then the Chapel compiler adds code to it.

These should probably say Chapel and the version.

``` chapel
// tt.chpl
proc test() {
  writeln("Hello");
}
test();
```

```
chpl tt.chpl --llvm  --llvm-print-ir test  --llvm-print-ir-stage full --gen-ids -g
```

```
!llvm.ident = !{!160}
!160 = !{!"clang version 6.0.0 (tags/RELEASE_600/final)"}
```

This could possibly be fixed by adding something to setupModule in `compiler/llvm/clangUtil.cpp`. I think that llvm.ident is an llvm::NamedMDNode. `get_version()` in version.h can be used to get the Chapel version string.

**Comment by mppf** [comment]
I've marked this as a good first issue. It's a good first issue for somebody wanting to modify the Chapel compiler and with some LLVM experience.

**Comment by mnafees** [comment]
Can we close this now @mppf?

**Comment by mppf** [comment]
Resolved by  #12450. Thanks @mnafees !
